server/api/models/Fundraise.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  attributes: {
+    campaignName: {
+      type: 'string',
+      required: true,
+    },
+    categoryName: {
+      type: 'string',
+      required: true,
+    },
+    categoryKey: {
+      type: 'string',
+      required: true,
+    },
+    amount: {
+      type: 'number',
+    },
+    givers: {
+      type: 'number',
+    },
+    milestones: {
```

Comment:
im assuming the usecase will be something like:
[
  {
    "value": "0",
    "description": "start"
  },
  {
    "value": "50",
    "description": "we have enough money to buy a car"
  },
  {
    "value": "100",
    "description": "fundraising goal"
  }
]

Reply:
yes, i added more comments and examples now to make it clear in the code